cmake/android/android_gradle_projects.cmake

```diff
@@ -1,7 +1,10 @@
 # https://developer.android.com/studio/releases/gradle-plugin
-set(ANDROID_GRADLE_PLUGIN_VERSION "3.2.1" CACHE STRING "Android Gradle Plugin version (3.0+)")
+set(ANDROID_GRADLE_PLUGIN_VERSION "4.0.1" CACHE STRING "Android Gradle Plugin version (4.0.1)")
 message(STATUS "Android Gradle Plugin version: ${ANDROID_GRADLE_PLUGIN_VERSION}")
 
+set(GRADLE_VERSION "6.1.1" CACHE STRING "Gradle version (6.1.1)")
```

Comment:
What prevent from using most recent `6.8.2` ?

Reply:
> What prevent from using most recent 6.8.2 ?

Nothing - I'm just proposing a PR to fix this one particular issue. If you want to propose a PR to bring the OpenCV Android implementation up to the bleeding edge (maybe add some Kotlin extensions as well) then please go ahead 😃